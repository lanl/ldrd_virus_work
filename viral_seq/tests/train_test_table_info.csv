table	Train_shape	Train_sum	Test_shape	Test_sum
main	(861, 186873)	52509595.08465634	(758, 221700)	37440755.00019514
kpc7	(861, 149872)	4399176.0	(758, 202193)	3705001.0
k5	(861, 193196)	1835719.0	(758, 287981)	1960625.0
k6	(861, 100109)	796230.0	(758, 153302)	891304.0
k7	(861, 65456)	647226.0	(758, 109083)	709050.0
k8	(861, 57444)	605026.0	(758, 96702)	643715.0
k9	(861, 52769)	577679.0	(758, 88584)	599034.0
k10	(861, 48835)	555769.0	(758, 81827)	563757.0
best_rfc	(861, 10693)	20684875.203283858	(758, 10693)	14331283.975410467

