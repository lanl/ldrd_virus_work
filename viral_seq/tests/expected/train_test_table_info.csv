table	Train_shape	Train_sum	Test_shape	Test_sum
main	(861, 442)	1222536.0846563554	(758, 442)	1040512.0001951418
kAA2	(861, 436)	6599756.0	(758, 423)	4598988.0
kAA3	(861, 8207)	6581453.0	(758, 8311)	4586719.0
kAA4	(861, 98316)	5463758.0	(758, 119896)	4254024.0
kAA5	(861, 193195)	1465489.0	(758, 287980)	1673722.0
kAA6	(861, 100108)	426000.0	(758, 153301)	604401.0
kAA7	(861, 65455)	276996.0	(758, 109082)	422147.0
kAA8	(861, 57443)	234796.0	(758, 96701)	356812.0
kAA9	(861, 52768)	207449.0	(758, 88583)	312131.0
kAA10	(861, 48834)	185539.0	(758, 81826)	276854.0
kPC2	(861, 64)	6599886.0	(758, 59)	4599031.0
kPC3	(861, 411)	6586953.0	(758, 394)	4587822.0
kPC4	(861, 2594)	6571154.0	(758, 2639)	4575892.0
kPC5	(861, 14829)	6503098.0	(758, 16098)	4549561.0
kPC6	(861, 61597)	6010786.0	(758, 73461)	4361303.0
kPC7	(861, 149871)	4028953.0	(758, 202192)	3418098.0
kPC8	(861, 190029)	1699719.0	(758, 274699)	1784303.0
kPC9	(861, 143165)	680068.0	(758, 204029)	828130.0
kPC10	(861, 90256)	360375.0	(758, 132919)	490323.0
best_rfc	(861, 7640)	16272360.605263904	(758, 7640)	11251002.201963443
